### PR TITLE
Fixed Legrand/Bticino LedInDark feature

### PR DESCRIPTION
**Pull request by FabianMangold**
This PR:
- Adds a missing FZ converter to handle Legrand / Bticino's "LedInDark" feature
- Adds the relevant configuration to the affected devices
- Removes Legrand's 067776_inverted switch

### CODE REVIEW

File: src/converters/fromZigbee.js [hunk 1]
```diff
@@ -3930,6 +3930,13 @@ const converters = {
             }
         },
     },
+    legrand_ledInDark: {
```
Comment (Koenkk) [inline src/converters/fromZigbee.js:3933]: Please rename to `legrand_led_in_dark`

File: src/devices/legrand.js [hunk 2]
```diff
@@ -153,31 +153,6 @@ module.exports = [
             exposes.binary('led_in_dark', ea.ALL, 'ON', 'OFF').withDescription(`Enables the LED when the power socket is turned off,
             allowing to see it in the dark`)],
     },
-    {
-        // Some require coverInverted:
-        // - https://github.com/Koenkk/zigbee2mqtt/issues/15101#issuecomment-1356787490
-        // - https://github.com/Koenkk/zigbee2mqtt/issues/16090
-        fingerprint: [
```
Comment (Koenkk) [inline src/devices/legrand.js:160]: Why was this removed?

Comment (FabianMangold) [reply src/devices/legrand.js]: That's correct, and it was not my intention to break anything.

Until shortly, this was required as there was no other alternative. Now that OTA is available, those devices work perfectly with the classic (non inverted) 067776.
I am preparing another PR with more extensive changes to the 067776, and wanted to avoid duplication.
The change has been reverted, hope that is OK.

**Comment by Koenkk** [comment]
thanks!
